xiaogpt/bot/edgegpt_bot.py

```diff
@@ -0,0 +1,47 @@
+from EdgeGPT import Chatbot, ConversationStyle
+from rich import print
+
+from xiaogpt.utils import split_sentences
+
+
+class EdgeGPTBot:
+    def __init__(self, cookie_file, conversation_style="creative", proxy=None):
+        self.history = []
+        self._bot = Chatbot(proxy=proxy, cookiePath=cookie_file)
+        self._wss_link = "wss://sydney.bing.com/sydney/ChatHub"
+        self._conversation_style = ConversationStyle.creative
```

Comment:
We can make it passed in by options instead of instance attributes, you can have default values like this:
https://github.com/yihong0618/xiaogpt/blob/994f5c553a58843ce25553d3dc5a4e513c424302/xiaogpt/bot/chatgptapi_bot.py#L23-L25